corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -1319,6 +1319,9 @@ hqDefine("cloudcare/js/formplayer/menus/views", [
             'keydown .js-home': 'onKeyActionHome',
         },
         onClickHome: function () {
+            if (!FormplayerFrontend.confirmUserWantsToNavigateAwayFromForm()) {
```

Comment:
Just to make sure i'm understanding, it looks like `setUnsavedFormInProgress` is called upon entering form and `unsavedFormInProgress` is set to True. `unsavedFormInProgress` would only be set to false if the user selects "home", "breadcrumb" or "clears form". 

It seems like if the user navigates away from the form in any other mean (like form submission, can't think if there's any other) then `unsavedFormInProgress` won't be set to false so on the next "home" or "breadcrumb" selection, they'll run into the confirmation window even if they aren't in a form?